modules/op-running-pipelines-in-restricted-environment.adoc

```diff
@@ -0,0 +1,204 @@
+// Module included in the following assemblies:
+//
+// pipelines/creating-applications-with-cicd-pipelines
+
+[id="op-running-pipelines-in-restricted-environment_{context}"]
+=  Running pipelines in a restricted environment
+
+
+To run OpenShift Pipelines in a disconnected cluster or a cluster provisioned in a restricted environment, you must ensure that either the Samples Operator is configured for a restricted network, or a cluster administrator has created a cluster with a mirrored registry.
+
+The following procedure uses the `Pipelines Tutorial` example to create a pipeline for an application in a restricted environment using a cluster with a mirrored registry. To ensure that the `pipelines-tutorial` example works in a restricted environment, you must mirror the respective builder images from the mirror registry for the front-end interface, `vote-ui`; back-end interface, `vote-api`; and the `cli`.
```

Comment:
Can `Pipelines Tutorial` here be just `pipelines-tutorial`? Looks like it's `pipelines-tutorial` everywhere else in the assembly.